.github/workflows/tests.yml

```
@@ -79,22 +127,20 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
         os: [ubuntu-latest]
         include:
           - python-version: '3.11'
             jupyter_server-version: '<2'
           - python-version: '3.11'
             jupyter_server-version: '>=2'
-          - python-version: '3.7'
-            os: windows-latest
-          - python-version: '3.11'
+          - python-version: '3.8'
```

This is what I meant:

```suggestion
          - python-version: '3.8'
            os: windows-latest
          - python-version: '3.11'
            os: windows-latest
```